example/plugins/lua-api/modsecurity/README.md

```diff
@@ -30,7 +30,7 @@ Contents/Rules inside example.conf
 
 Working with CRS
 ====
- - Go to [here](https://github.com/SpiderLabs/owasp-modsecurity-crs) and get release v3.2.0
+ - Go [here](https://github.com/coreruleset/coresuleset) and download release v3.3.2
```

Comment:
is this the right repo URL? I get a 404 visiting that.

Reply:
fixed the typo